webapp/components/CommentList/CommentList.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <div id="comments" class="comment-list">
     <h3 class="title">
-      <counter-icon icon="comments" :count="postComments.length" />
+      <counter-icon
+        icon="comments"
+        :count="postComments.filter((comment) => !comment.deleted).length"
```

Comment:
I looked at the other place in `webapp/components/PostTeaser/PostTeaser.vue` where the counter is set on a post teaser. This is the part which differs from that here.
Because I know this value of `post.commentsCount` comes direct from the backend and because I have found this doesn’t count `comment.deleted` or `comment.disabled` I will suggest this solution to be taken over here.

```suggestion
        :count="post.commentsCount“
```

A test in `backend/src/schema/resolvers/posts.spec.js` is missing which checks that only comments not deleted and not disabled are counted by the Cypher queries.
If you are motivated would be nice to implement this one. 💫🍀

Reply:
Thanks a lot, this one seems a lot better than a filter on the deleted comments!
I will implement this so the frontend code stays clean logic.